tests/test_pretty_print.py

```diff
@@ -475,7 +475,7 @@ def main():
     builddir = ezsearch('--builddir=(.*?) ', ' '.join(sys.argv) + ' ')
     if not builddir or not os.path.exists(builddir):
         builddir = os.getcwd()
-        print('warning: couldnt find --builddir, trying to use current dir:', builddir)
+        print('warning: couldn't find --builddir, trying to use current dir:', builddir)
```

Comment:
This causes the tests to fail as the outer quotes are single `'` too.

Reply:
a6a5f62